packages/cardhost/app/components/code-editor.js

```diff
@@ -22,13 +22,25 @@ requirejsPolyfill();
 export default class CodeEditor extends Component {
   editor; // value is set by renderEditor
 
+  constructor(...args) {
+    super(...args);
+
+    this.resizable = (this.args.resizable === true) ? true : false;
```

Comment:
```suggestion
    this.resizable = (this.args.resizable === true) ? true : false;
```

In general, I've done these kinds of assignments with getters, just as a pattern. Using the constructor works fine too.